.plans/feat-471/errors/backend-pipeline.md

```
@@ -0,0 +1,146 @@
+Run # Create the database directory
+  # Create the database directory
```

markdownlint-cli2 (0.18.1)

Headings must start at the beginning of the line

(MD023, heading-start-left)